src/man/firejail.1.in

```diff
@@ -983,23 +983,6 @@ Example:
 .br
 $ firejail \-\-net=eth0 \-\-ip=10.10.20.56 firefox
 
-.TP
-\fB\-\-ip=none
-No IP address and no default gateway are configured for the last interface
-defined by a \-\-net option. Use this option
-in case you intend to start an external DHCP client in the sandbox.
-.br
-
-.br
-Example:
-.br
-$ firejail \-\-net=eth0 \-\-\ip=none
-.br
-
-.br
-If the corresponding interface doesn't have an IP address configured, this
-option is enabled by default.
-
```

Comment:
This also applies to firejail-profile.5.

The items in both man pages are more or less the same, so it's important to
always update them in sync to avoid making their contents inconsistent with
each other.

Reply:
Fixed.